README.md

```diff
@@ -147,6 +147,17 @@ Here is a list of variables that you can use to customize your newly copied `.en
 | PNY | `pny`|
 | Zotac | `zotac`|
 
+#### Supported Brands and Models
+| **Brand** | **Model 1** | **Model 2** | **Model 3** | **Model 4** | **Model 5** | **Model 6** |
+|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| asus | rog strix | rog strix oc | strix | tuf | tuf oc | |
+| evga | ftw3 | ftw3 ultra | xc3 black | xc3 | xc3 ultra | |
+| gigabyte | eagle | eagle oc | gaming | gaming oc | | |
+| msi | gaming x trio | ventus 3x | ventus 3x oc | | | |
+| nvidia | founders edition | | | | | |
+| pny | dual fan | xlr8 | xlr8 rgb | | | |
+| zotac | trinity | trinity oc | | | | |
```

Comment:
```suggestion
#### Supported Brands and Models
| **Brand** | **Model** |
|:---:|---|
| asus | rog strix, rog strix oc, strix, tuf, tuf oc |
| evga | ftw3, ftw3 ultra, xc3 black, xc3, xc3 ultra |
| gigabyte | eagle, eagle oc, gaming, gaming oc |
| msi | gaming x trio, ventus 3x, ventus 3x oc |
| nvidia | founders edition |
| pny | dual fan, xlr8, xlr8 rgb |
| zotac | trinity, trinity oc |
```

I think format will be better since they can all have different amounts.

Reply:
That works and no, glad to be a tiny tiny bit of help haha!

Reply:
It's more than a tiny! Thank you very much.